packages/sync/src/links/OfflineQueueLink.ts

```diff
@@ -121,10 +120,16 @@ export class OfflineQueueLink extends ApolloLink {
     return new Promise((resolve, reject) => {
       opEntry.subscription = forward(operation).subscribe({
         next: result => {
+          if (this.listener && this.listener.onOperationSuccess) {
```

Comment:
[Oftopic] Optional chaining in typescript will transpile that. More info https://medium.com/inside-rimeto/optional-chaining-in-typescript-622c3121f99b